packages/ui-components/src/Type/Text.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import classnames from "classnames/bind";
+
+import styles from "./Text.module.scss";
+import objectToClassNames from "../utils/objectToClassnames";
+
+interface OwnTextProps {
+  type?: TextType;
+  noWrap?: boolean;
+}
+
+export type TextProps = React.HTMLAttributes<HTMLSpanElement> & OwnTextProps;
+
+export type TextType =
+  | "body"
+  | "body-strong"
+  | "caption"
+  | "caption-strong"
+  | "code";
+
+const cx = classnames.bind(styles);
+
+export const Text = ({
+  type = "body",
+  noWrap = false,
+  className,
+  children,
+  ...rest
+}: TextProps) => {
+  const classNames = cx(
+    objectToClassNames({ type }),
+    { ".no-wrap": noWrap },
```

Comment:
".no-wrap" class name should be provided here without dot, it's not recognized properly.
Would you mind to extend Demo storybook with `nowrap` option?

Reply:
Good catch on both counts. I thought the `noWrap` prop was a nice convenience but then forgot to document it in Storybook.